pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.51</version>
+        <version>4.63</version>
```

Comment:
Latest version at the time of this writing. Requires 2.361.1 or later and Java 11.